### DISCUSSION TITLE
Remote access?

### DISCUSSION THREAD

**Discussion by duringleaves** [bigsk1/voice-chat-ai#8] [General]
I've got voice-chat-ai installed on my server, but when I try to use it from my desktop, it fails as it appears to be looking for local input and output audio devices.  Is it not possible to use it over the LAN from a different computer?

**Comment by bigsk1** [comment]
not currently but maybe in docker, it needs your microphone local access and permissions and not using it via browser, something I can look at later, might be CORS issue also.

**Comment by bigsk1** [comment]
@duringleaves just did a big update might work now if volume mapped audio devices are correct. Added docker support.
